www/kukd01/homework-08/js/app.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Git The Hub
+ */
+
+var App = App || {};
+// INSERT CLIENT ID FROM GITHUB
+App.client_id = '248e90c1e8e75a696626';
+// INSERT CLIENT SECRET FROM GITHUB
+App.client_secret = 'ac7652120d5103e5ea1a3612aeec262644006cb4';
+App.baseApiUrl = 'https://api.github.com';
+App.searchValue = $('#search-input');
```

Comment:
tohle mas dvakrat, tady a v `App.init`

Reply:
To jsem zapomněl, když jsem to chtěl definovat globálně :)